mapie/multi_label_classification.py

```diff
@@ -699,7 +601,6 @@ def fit(
     def predict(
         self,
         X: ArrayLike,
-        method: Optional[str] = "crc",
         alpha: Optional[Union[float, Iterable[float]]] = None,
         delta: Optional[float] = None,
         bound: Optional[Union[str, None]] = "wsr"
```

Comment:
By default, MapieMultilabelClassifier controls the recall with CRC, hence by default I think bound should be None, as if a user uses this class with all the default parameters, he shouldn't have any warning

Reply:
Done, I have change the doc so that user know when method is rcps and bound is None the bound use in crc_rcps file is wsr.